projects/packages/search/src/class-plan.php

```diff
@@ -122,19 +122,33 @@ public function update_search_plan_info( $response ) {
 		$body        = json_decode( wp_remote_retrieve_body( $response ), true );
 		$status_code = wp_remote_retrieve_response_code( $response );
 
-		if ( 200 !== $status_code || ! isset( $body['supports_instant_search'] ) ) {
+		if ( 200 !== $status_code ) {
+			return null;
```

Comment:
It looks like the return signature for this function is `true | null`. Would it make more sense for it to be `bool`, where true signifies a successful execution and false signifies a noop?

Reply:
Good point. Fixed in c1ed321a1.